hw/top_earlgrey/BUILD

```diff
@@ -129,6 +129,26 @@ fpga_cw310(
     rsa_key = {"//sw/device/silicon_creator/rom_ext/keys/fake:rom_ext_test_private_key_0": "test_key_0"},
 )
 
+fpga_cw310(
+    name = "fpga_hyper310_test_rom",
```

Comment:
Is there a compelling reason to add a test ROM environment for hyper310 and not just use `//hw/top_earlgrey:fpga_cw310_sival_rom_ext` or `//hw/top_earlgrey:fpga_hyper310_rom_ext`?

If not, it'd be nice if we didn't expand the maintenance surface, and the changes to this file and rules/opentitan/defs.bzl can be removed.

Reply:
Short answer: no.

Long answer: I was originally seeing test failures on cw310 that I believed were a result of a cw310 issue, so I moved this test over to hyper310. However, the issues eventually showed up on hyper310 as well and they were traced to a Tock driver bug. Now that the bug is fixed, we can switch back to cw310.

I don't think Tock will run with a ROM EXT yet, so those particular environments aren't an option.